lib/std/os/windows.zig

```diff
@@ -1569,6 +1569,54 @@ pub fn sliceToPrefixedFileW(s: []const u8) !PathSpace {
     return path_space;
 }
 
+/// Convert `dos_path` to an NT path that can be passed to Nt* functions like NtCreateFile.
+pub const NtPath = struct {
+    str: UNICODE_STRING,
+    pub fn initA(dos_path: []const u8) !NtPath {
```

Comment:
The "A" suffix is misleading when the function accepts utf8.

Reply:
It's the standard windows mechanism `A` vs `W`.  The implementation may utilize the `utf8` function to convert it to a u16 array, but the OS only accepts Ascii characters AFAIK.

Reply:
I've removed this method altogether and replaced it by providing a function that any code can use to convert a UTF8 encoded path to a Wide Character windows path on the stack `utf8ToWPathSpace`.